cmd/ebpf_exporter/main.go

```diff
@@ -14,9 +14,10 @@ import (
 )
 
 func main() {
-	listenAddress := kingpin.Flag("web.listen-address", "The address to listen on for HTTP requests").Default(":9435").String()
-	configFile := kingpin.Flag("config.file", "Config file path").Default("config.yaml").File()
-	debug := kingpin.Flag("debug", "Enable debug").Bool()
+	configFile := kingpin.Flag("config.file", "Config file path.").Default("config.yaml").File()
+	debug := kingpin.Flag("debug", "Enable debug.").Bool()
```

Comment:
Let's not have dots in the description. LGTM otherwise.

Reply:
fixed